### PR TITLE
[May18] Surface Magnetism + Spatial Processing example scene

### PR DESCRIPTION
**Pull request by cre8ivepark**
Overview
---
Adding an example scene which shows how to make objects smoothly align with the surfaces using SolverSurfaceMagnetism. Based on existing SpatialProcessing example scene, added a plane with MRTK logo with SolverSurfaceMagnetism script. 

Demo video
---
https://gfycat.com/PitifulRareKodiakbear

Changes
---
- No code changes or updates. 
- Added new scene SpatialProcessingSurfaceMagnetism.unity

Readme
---
[SpatialProcessingSurfaceMagnetism.md](https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/may18_dev_SurfaceMagnetism/Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md)

### CODE REVIEW

File: Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat [hunk 1]
```diff
@@ -72,7 +73,7 @@ Material:
     - _WireThickness: 100
     - _ZWrite: 1
     m_Colors:
-    - _BaseColor: {r: 0, g: 0, b: 0, a: 1}
+    - _BaseColor: {r: 0.0036224083, g: 0.24982113, b: 0.49264705, a: 1}
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat]: Should we be editing the base material, or using a copy for our example?

May need to revert.

Comment (davidkline-ms) [reply Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat]: Agreed. Why was the color changed?

Comment (cre8ivepark) [reply Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat]: Oh, I wanted to update default white to more similar to what we have in Fragments (changed it to blue)
Original:
2018-05-17 10_05_49-photos
Updated:
2018-05-17 10_06_13-photos

Comment (cre8ivepark) [reply Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat]: I can duplicate with a new name 'WireframeBlue.mat' if we want. and assign it to SpatialMapping prefab as a default material.

Comment (davidkline-ms) [reply Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat]: Please do duplicate the material. Changing the color can cause confusion with customers.

Thanks!

**Comment by cre8ivepark** [comment]
@davidkline-ms Reverted Wireframe.mat to black & white. and added WireframeBlue.mat. Thanks!

**Comment by cre8ivepark** [comment]
Applied same updates to the PR for Dev branch #2114

**Pull request merged**